Ambiguous-Squids/bangazon-api
# As cleanapi

Updated the Bangazon API to allow an Angular client to consume

## Description
Added CORS app and tested to ensure that a client can consume API

## Related Issue
#4 

## Motivation and Context
Prior to using the django CORS app, you could not consume this REST API from a client using AJAX calls

## How Has This Been Tested?
`pip install django-cors-headers`
`./manage.py migrate`
`./manage.py runserver`

## Steps to Test
`pip install django-cors-headers`
`./manage.py migrate`
`./manage.py runserver`


System configuration.
3rd party libraries that need to be installed.
Command line utilities to run.
If there is a UI component, give clear instructions for steps to perform in the UI, and what they should expect to see as the outcome of those steps.

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help!
